### ISSUE TITLE
Scrollable elements on page exercises-oss-jekyll are not keyboard accessible 

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#715]
**Description**
Elements that have scrollable content should be accessible by keyboard

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/exercises-oss-jekyll.html
*CSS Selector(s) (if possible and applicable)*: 
 ```.language-bash.highlighter-rouge:nth-child(79) > .highlight > pre > .nginx.hljs```
```.language-bash.highlighter-rouge:nth-child(32) > .highlight > pre > .coffeescript.hljs```
```.language-bash.highlighter-rouge:nth-child(52) > .highlight > pre > .xml.hljs```
```.language-bash.highlighter-rouge:nth-child(54) > .highlight > pre > .coffeescript.hljs```
```.javascript```
```.language-bash.highlighter-rouge:nth-child(62) > .highlight > pre > .bash.hljs```
```.language-bash.highlighter-rouge:nth-child(64) > .highlight > pre > .bash.hljs```
```.language-bash.highlighter-rouge:nth-child(66) > .highlight > pre > .css.hljs```
```.language-bash.highlighter-rouge:nth-child(71) > .highlight > pre > .nginx.hljs```
```.language-bash.highlighter-rouge:nth-child(73) > .highlight > pre > .nginx.hljs```
```.language-bash.highlighter-rouge:nth-child(79) > .highlight > pre > .nginx.hljs```
```.language-bash.highlighter-rouge:nth-child(81) > .highlight > pre > .nginx.hljs```

*Additional Identifier(s) (if possible and applicable)*:  ```<code class="hljs nginx">```
*Additional Details*:  

**Screenshot(s) (if possible)**

Here is an example

<img width="522" alt="Screen Shot 2020-01-30 at 2 27 18 PM" src="https://user-images.githubusercontent.com/26819992/73482954-a6ab6200-436c-11ea-81c7-d87456ec1aa5.png">

**Additional information or thoughts**

Fix at least one (1) of these issues:
- Element should have focusable content
- Element should be focusable

As a result of scan #675
